velox/functions/lib/aggregates/tests/utils/AggregationTestBase.cpp

```diff
@@ -633,9 +665,9 @@ void AggregationTestBase::testReadFromFiles(
   auto writerPool = rootPool_->addAggregateChild("AggregationTestBase.writer");
 
   // Splits and writes the input vectors into two files, to some extent,
-  // involves shuffling of the inputs. So only split input if allowInputShuffle_
-  // is true. Otherwise, only write into a single file.
-  if (allowInputShuffle_ && input->size() >= 2) {
+  // involves shuffling of the inputs. So only split input if aggregate
+  // is non-orderSensitive. Otherwise, only write into a single file.
+  if (!hasOrderSensitive(aggregates) && input->size() >= 2) {
```

Comment:
@yanngyoung  Parsing of SQL expressions in hasOrderSensitive is duplicative and error prone. Instead, fetch the function names from the builder.planNode() (cast to AggregationNode first).

Reply:
updated, please have a look, thanks!